pydis_core/exts/source.py

```diff
@@ -160,6 +161,8 @@ def _get_source_link(self, source_item: object, source_type: _SourceType) -> tup
 
         if source_type == _SourceType.core_command or source_type == _SourceType.core_cog:
             version = f"v{metadata.version('pydis_core')}"
+        elif sha := os.getenv("GITHUB_SHA"):
```

Comment:
this could be provided as a setup variable with a default of "main"

Reply:
It could, but this works with our current bots without change on their side. Just seems easier and is guaranteed to work for our use cases.

Setup for the bot core instance is already massive.